vaadin-dir-mixin.html

```diff
@@ -0,0 +1,97 @@
+<script>
+(function() {
+  /**
+   * Array of Vaadin custom element classes that have been subscribed to the dir changes.
+   */
+  const directionSubscribers = [];
+  const directionUpdater = function() {
+    if (directionSubscribers) {
+      directionSubscribers.forEach(element => {
+        if (document.documentElement.getAttribute('dir')) {
```

Comment:
This check could be moved outside `forEach`:

```js
const dir = document.documentElement.getAttribute('dir');
```
So you only get it once instead of doing it for each subscriber.

Reply:
Updated.